app/Controllers/importExportController.php

```diff
@@ -451,15 +474,13 @@ private function importJson(string $article_file, bool $starred = false): bool {
 			}
 			$newGuids[$entry->guid()] = true;
 
-			/** @var FreshRSS_Entry|null */
 			$entry = Minz_ExtensionManager::callHook('entry_before_insert', $entry);
-			if ($entry == null) {
+			if ($entry === null) {
```

Comment:
Especially since this will be a response from third-party code, it is better to be on the safe side, so either keep the safer `==` or (maybe overkill) test for object / FreshRSS_Entry.

```suggestion
			if ($entry == null) {
```